doc/refman/RefMan-uti.tex

```diff
@@ -108,27 +108,220 @@
+  sorted table of the per-line timing differences between the timing
+  logs in the \texttt{BEFORE} and \texttt{AFTER} files, display it,
+  and save it to the file specified by the
+  \texttt{TIME\_OF\_PRETTY\_BUILD\_FILE} variable, which defaults to
+  \texttt{time-of-build-pretty.log}.
+
+  To generate the \texttt{.v.before-timing} or
+  \texttt{.v.after-timing} files, you should pass
+  \texttt{TIMING=before} or \texttt{TIMING=after} rather than
+  \texttt{TIMING=1}.
+
+  \texttt{Note}: The sorting used here is the same as in the
+  \texttt{print-pretty-timed-diff} target.
+
+  \texttt{Note}: This target requires \texttt{python} to build the table.
+
+  \texttt{Example}: For example, running
+  \texttt{print-pretty-single-time-diff} might give a table like this:
+\begin{verbatim}
+After     | Code                                                | Before    || Change    | % Change
+---------------------------------------------------------------------------------------------------
+0m00.50s  | Total                                               | 0m04.17s  || -0m03.66s | -87.96%
+---------------------------------------------------------------------------------------------------
+0m00.145s | Chars 069 - 162 [Definition~foo0~:=~Eval~comp~i...] | 0m00.192s || -0m00.04s | -24.47%
+0m00.126s | Chars 000 - 026 [Require~Coq.ZArith.BinInt.]        | 0m00.143s || -0m00.01s | -11.88%
+   N/A    | Chars 027 - 068 [Declare~Reduction~comp~:=~nati...] | 0m00.s    || +0m00.00s | N/A
+0m00.s    | Chars 027 - 068 [Declare~Reduction~comp~:=~vm_c...] |    N/A    || +0m00.00s | N/A
+0m00.231s | Chars 163 - 208 [Definition~foo1~:=~Eval~comp~i...] | 0m03.836s || -0m03.60s | -93.97%
+\end{verbatim}
+
+\item \texttt{all.timing.diff}, \texttt{path/to/file.v.timing.diff}
+  --- The \texttt{path/to/file.v.timing.diff} target will make a
+  \texttt{.v.timing.diff} file for the corresponding \texttt{.v} file,
+  with a table as would be generated by the
+  \texttt{print-pretty-single-time-diff} target; it depends on having
+  already made the corresponding \texttt{.v.before-timing} and
+  \texttt{.v.after-timing} files, which can be made by passing
+  \texttt{TIMING=before} and \texttt{TIMING=after}.  The
+  \texttt{all.timing.diff} target will make such timing difference
+  files for all of the \texttt{.v} files that the \texttt{Makefile}
+  knows about.  It will fail if some \texttt{.v.before-timing} or
+  \texttt{.v.after-timing} files don't exist.
+
+  \texttt{Note}: This target requires \texttt{python} to build the table.
+\end{itemize}
+
 \paragraph{Notes about including the generated Makefile}
 
 This practice is discouraged. The contents of this file, including variable names
 and status of rules shall change in the future.  Users are advised to
 include {\tt Makefile.conf} or call a target of the generated Makefile
 as in {\tt make -f Makefile target} from another Makefile.
 
+One way to get access to all targets of the generated
```

Comment:
This section is good, and I hope I'll manage to integrate with more (to come) doc for coq-makefile.
But has little to do in this PR.